adapter/src/main/java/com/twilio/kudu/sql/parser/SqlCreateMaterializedView.java

```diff
@@ -35,14 +36,17 @@ public class SqlCreateMaterializedView extends SqlCall {
   public final SqlIdentifier cubeName;
   public final boolean ifNotExists;
   public final SqlSelect query;
+  public final SqlNodeList orderByList;
 
   /** Creates a SqlCreateView. */
-  SqlCreateMaterializedView(SqlParserPos pos, SqlIdentifier cubeName, boolean ifNotExists, SqlSelect query) {
+  SqlCreateMaterializedView(SqlParserPos pos, SqlIdentifier cubeName, boolean ifNotExists, SqlSelect query,
+      SqlNodeList orderByList) {
```

Comment:
will the kudu range partitions also be setup based on this column?
do we need changes to kk while setting up the partitioning?

Reply:
I pointed out in a separate comment where how the range partitioning of cubes are setup.